lib/que/job.enqueue_spec.rb

```diff
@@ -122,6 +122,21 @@ def assert_enqueue(
       expected_priority: 4
   end
 
+  it "should be able to use an explicit `job_options` keyword to avoid conflicts with job keyword args" do
+    assert_enqueue \
+      [1, {string: "string", priority: 10, job_options: { priority: 15 }}],
+      expected_args: [1, {string: "string", priority: 10}],
+      expected_priority: 15
+  end
+
+  it "should fall back to using job options specified at the top level if not specified in job_options" do
+    assert_enqueue \
+      [1, {string: "string", run_at: Time.now + 60, priority: 10, job_options: { priority: 15 }}],
+      expected_args: [1, {string: "string", priority: 10}],
+      expected_run_at: Time.now + 60,
+      expected_priority: 15
+  end
```

Comment:
The title and the expectation here don't quite match.

You are passing the `priority` in via the both the `job_options` and keyword args.  I would have expected the priority to be missing from `job_options` and the `expected_priority` in the assertion to be 10.

Reply:
ahh it is the `run_at` option that is being tested here.  Sorry the priority appearing in both places got me messed up